libs-scala/oracle-testing/src/main/scala/testing/oracle/OracleAroundAll.scala

```diff
@@ -15,6 +15,7 @@ trait OracleAroundAll extends OracleAroundSuite with BeforeAndAfterAll {
   }
 
   override protected def afterAll(): Unit = {
+    dropUser()
```

Comment:
I am already taking care of this, if that is ok with you. BTW this needs to go after the super.afterAll

Reply:
sure, I recommend factoring out the fix to the fixture from whatever other changes you’re making.